### PR TITLE
Renamecontribute

### PR DESCRIPTION
**Pull request by nityaravi**
**Release Note Template for Downstream PRs (will be copied)**

See [Write release notes](https://googlecloudplatform.github.io/magic-modules/contribute/release-notes/) for guidance.

```release-note:none

```

### CODE REVIEW

**Review by melinath — CHANGES_REQUESTED** [review]
could you resolve the merge conflict?

**Comment by nityaravi** [comment]
> could you resolve the merge conflict?

Done

File: docs/content/reference/field.md [hunk 1]
```diff
@@ -4,6 +4,7 @@ weight: 20
 aliases:
   - /reference/field-reference
   - /develop/field-reference
+  - /reference/field-reference
```
Comment (melinath) [inline docs/content/reference/field.md:7]:
```suggestion
```

File: docs/content/reference/resource.md [hunk 2]
```diff
@@ -5,6 +5,7 @@ aliases:
   - /reference/resource-reference
   - /reference/iam-policy-reference
   - /develop/resource-reference
+  - /reference/resource-reference
```
Comment (melinath) [inline docs/content/reference/resource.md:8]:
```suggestion
```

**Review by melinath — APPROVED** [review]

**Review by melinath — APPROVED** [review]

**Pull request merged**